romancal/pipeline/exposure_pipeline.py

```diff
@@ -155,6 +153,8 @@ def process(self, input):
                 result = self.flatfield(result)
                 result = self.photom(result)
                 result = self.source_detection(result)
+                if file_type == "asdf":
+                    result = self.tweakreg(result)
```

Comment:
We'll never get to this point with `file_type = "asdf"`, since we're always setting it to `"asn"` regardless of the type of input.